src/sentry/models/organization.py

```diff
@@ -198,6 +198,9 @@ class flags(TypedClassBitField):
         # Disable org-members from creating new projects
         disable_member_project_creation: bool
 
+        # Disable superuser access to an organization
+        disable_superuser_access: bool
```

Comment:
This is a negative since the default is False and we **do not** want to have data secrecy enabled by default.